src/UI/Internal/Dialog.elm

```diff
@@ -117,7 +114,7 @@ titleText cfg padding title =
     Text.heading5 title
         |> Text.renderElement cfg
         |> Element.el
-            [ Element.width fill
+            [ Element.width (px 400)
```

Comment:
again if this is meant for mobile, might not fit on iphones

Reply:
this wasn't supposed to be committed, oops